libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelBinder.kt

```diff
@@ -53,6 +54,17 @@ abstract class InfoPanelBinder : UIBinder {
      */
     abstract fun getContentLayout(layout: ViewGroup): ViewGroup?
 
+    /**
+     * Called when the Info Panel layout should apply system bar insets.
+     *
+     * This method should be overridden by a subclass to apply a policy different from the default behavior.
+     * The default behavior ignores insets.
+     *
+     * @param layout ViewGroup returned by [onCreateLayout]
+     * @param insets system bars insets
+     */
+    open fun applySystemBarsInsets(layout: ViewGroup, insets: Insets) = Unit
```

Comment:
Does this mean if as a developer, I want to use the InfoPanel from Drop-In UI, but want to draw it behind the navigation bar, I would have to subclass `InfoPanelBinder` and apply insets in this function? In this case, in my subclass `onCreateLayout`, `getHeaderLayout`, `getContentLayout` will all use the default values provided by Drop-In UI. 

If my understanding above it correct, I think there is a scope to improve this. As a developer subclassing `InfoPanelBinder` makes sense to me when I have my own `getHeadlerLayout`, `getContentLayout` etc. However, just to be able to apply `systemInsets` asking customers to subclass `InfoPanelBinder` is too much.

Reply:
> Does this mean if as a developer, I want to use the InfoPanel from Drop-In UI, but want to draw it behind the navigation bar, I would have to subclass InfoPanelBinder and apply insets in this function?

Not entirely. The default Info Panel Binder (MapboxInfoPanelBinder) already handles system bar insets and renders its contents behind the navigation bar. Developers will only have to subclass InfoPanelBinder if they want to provide their own layout.